src/app/event/[eventId]/(eventId)/page.tsx

```diff
@@ -16,38 +16,38 @@ import { IEvent } from '@/types/event'
 const CHIP_STYLE = '!font-bold !p-5 !border-none'
 const CHIP_STYLE_ACTIVE = CHIP_STYLE + ' ' + '!bg-teal-600 !text-teal-50'
 const CHIP_STYLE_INACTIVE = CHIP_STYLE + ' ' + '!bg-teal-50 !text-teal-400'
-type ChipType = 'plan' | 'timeline' | 'tracker' | 'rundown';
+type ChipType = 'plan' | 'timeline' | 'tracker' | 'rundown'
 
 function getAvatarComponent(type: string, chipType: ChipType): JSX.Element {
-  let avatarComponent;
+  let avatarComponent
 
   if (type === 'plan') {
     avatarComponent = (
       <AppRegistration
         className={`${chipType === type ? '!text-teal-50' : '!text-black'}`}
       />
-    );
+    )
   } else if (type === 'timeline') {
     avatarComponent = (
       <EditCalendar
         className={`${chipType === type ? '!text-teal-50' : '!text-black'}`}
       />
-    );
+    )
   } else if (type === 'tracker') {
     avatarComponent = (
       <PlaylistAdd
         className={`${chipType === type ? '!text-teal-50' : '!text-black'}`}
       />
-    );
+    )
   } else {
     avatarComponent = (
       <AssignmentIcon
         className={`${chipType === type ? '!text-teal-50' : '!text-black'}`}
       />
-    );
+    )
   }
```

Comment:
I really like your implementation. However, if you are dealing with more than 3 condition, in this case 4 condition, using case is always better alternative than if else. Not only it will help the code readability, also the next developer wouldn't have a hard time scrolling through your if condition. This is just suggestion though, feel free to disagree.

https://www.codecademy.com/forum_questions/5138c726679aea7b29005315#:~:text=The%20first%20of%20which%20is,a%20lot%20easier%20to%20read.

Reply:
Could also refactor the methos like

`function getAvatarComponent(type, chipType) {
  switch (type) {
    case 'plan':
      return <AppRegistration />;
    case 'timeline':
      return <EditCalendar />;
    case 'tracker':
      return <PlaylistAdd />;
    default:
      return <AssignmentIcon />;
  }
} `

for better readability. Just a suggestion though. Rest of the codes LGTM

Reply:
Have modified! Will merge soon😃